portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -22,22 +24,35 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/literature")
+/** Servlet that maintains the comment section.*/
+@WebServlet("/comment")
 public final class DataServlet extends HttpServlet {
 
-  private List<String> BookRecs;
+  private CommentHistory comments = new CommentHistory();
+
   @Override
-  public void init() {
-    BookRecs = new ArrayList<>();
-    BookRecs.add("The Thread by Victoria Hislop");
-    BookRecs.add("The Hunchback of Notre Dame by Victor Hugo");
-    BookRecs.add("The Brothers Karamazov by Fyodor Dostoevsky");
+  /** Load all comment history onload **/
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("application/json");
+    String json = new Gson().toJson(comments);
```

Comment:
As commented earlier, see if this can be made once and then used again and again.